### ISSUE TITLE
[BUG] Infinite page loading when using Chrome browser in developer mode

### ISSUE THREAD

**Issue by flexstat** [bunkerity/bunkerweb#1018]
### What happened?

When using any type of antibot challenge in the Google Chorome browser, when switching to developer mode and refreshing the page, the page load goes into an endless loading process that ends with 429 too many requests.

### How to reproduce?

You need to enable any anti-bot challenge, then go to the Google Chrome browser, go to the page with the challenge and then go to developer mode in the browser and refresh the page

### Relevant log output

```shell
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
bunkerweb_1        | 2024/04/03 10:17:29 [warn] 106#106: *23 [UTILS] session check failed : USER_AGENT!=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36, client:  , server: www.example.com, request: "GET /check HTTP/1.1", host: " "
```

### BunkerWeb version

1.5.6

### What integration are you using?

Docker

### Removed private data

- [X] I have removed all private data from the configuration file and the logs

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fl0ppy-d1sk** [comment]
Hello @flexstat,

That one was tricky. It's and edge bug when User-Agent used to generate the initial cookie is different when coming back with a new cookie. And we had an infinite loop... It will be fixed in 1.5.7 !

Thanks.

**Comment by fl0ppy-d1sk** [comment]
Hello @flexstat,

You can try the `testing` the version if you want and tell us if it fixes the issue : https://docs.bunkerweb.io/testing/

**Comment by fl0ppy-d1sk** [comment]
Hey @flexstat,

It's fixed in 1.5.7 ! Thanks.
